src/pages/billing/SpendReport/SpendReport.test.ts

```diff
@@ -158,6 +158,29 @@ describe('SpendReport', () => {
     // is in the integration test. Accessibility is also tested in the integration test.
   })
 
+  it('displays Azure cost information but does not include per-workspace costs', async () => {
+    //Arrange
+    const getSpendReport = jest.fn()
+    asMockedFn(Ajax).mockImplementation(() => ({
+      Billing: { getSpendReport } as Partial<AjaxContract['Billing']>
+    } as Partial<AjaxContract> as AjaxContract))
+    getSpendReport.mockResolvedValue(createSpendReportResult('1110'))
+
+    // Act
+    await act(async () => {
+      await render(h(SpendReport, { viewSelected: true, billingProjectName: 'thrifty', cloudPlatform: 'AZURE' }))
+    })
+
+    // Assert
+    await waitFor(() => {
+      expect(screen.getByText(/\$89.00 in other infrastructure/i)).toBeInTheDocument()
+    })
+    expect(getSpendReport).toHaveBeenCalledWith({ billingProjectName: 'thrifty', endDate: '2022-04-01', startDate: '2022-03-02', aggregationKeys: ['Category'] })
+    expect(screen.getByTestId('spend')).toHaveTextContent('$1,110.00*')
+    expect(screen.getByTestId('compute')).toHaveTextContent('$999.00')
+    expect(screen.getByTestId('storage')).toHaveTextContent('$22.00')
```

Comment:
Given the naming of this test I'd expect a negative assertion that the workspace costs are in fact excluded.

Reply:
We do verify that the workspace cost graph isn't present in the [integration test](https://github.com/DataBiosphere/terra-ui/blob/6a18a62d1a297ff4aaaf576244f30b281a4d9038/integration-tests/tests/billing-projects.js#L287). Here we're also validating that only `Category` costs are retrieved from Rawls. I assume we could also verify that there's no highcharts content present on the page, but I haven't looked into it